test/run_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -veu
+
+ROOT_DIR=$(pwd)
+
+if [[ "$(docker images -q usagi/test:latest 2> /dev/null)" == "" ]]; then
+    docker build -f Dockerfile-forTest -t usagi/test .
+fi
+
+docker run -it --rm \
+    -v ${ROOT_DIR}/test:/usagi/test \
+    -v ${ROOT_DIR}/api:/usagi/api \
+    -v ${ROOT_DIR}/importer:/usagi/importer \
+    -v ${ROOT_DIR}/installer:/usagi/installer \
+    -v ${ROOT_DIR}/command:/usagi/command \
+    -v ${ROOT_DIR}/config:/usagi/config \
```

Comment:
if possible, I would recommend that making `src` directory to keep our own code and mount it here so that we won't have to specify new directories.

Reply:
i.e., move all our own code (e.g., `api/`, `importer/`) to `src`. 

Something like:
```
usagi
├── Dockerfile
├── Dockerfile-forTest
├── Jenkinsfile
├── LICENSE
├── README.md
├── src/
├── test/
...
└── lib/
```